docs/migrating_2.rst

```diff
@@ -264,6 +265,26 @@ Guild.get_member taking positional only argument
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 :meth:`Guild.get_member` method's first argument is now positional only.
 
+Intents.message_content must be enabled to receive message content
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In order to receive message content in most messages, you must have :attr:`Intents.message_content` enabled.
+
+You can do this by using :meth:`Intents.all` or by setting :attr:`~Intents.message_content` to ``True``:
+
+.. code-block:: python3
+
+    intents = nextcord.Intents.default()
+    intents.message_content = True
+
+For more information go to the :ref:`message content intent documentation <need_message_content_intent>`.
+
+If you are unable to get the message content intent and would like some time to migrate to application commands:
```

Comment:
If you are unable to get the message content intent and would like some time to migrate to application commands you can downgrade the API back to version 9 as shown below.